config/initializers/diaspora_federation.rb

```diff
@@ -0,0 +1,8 @@
+# configure the federation engine
+DiasporaFederation.configure do |config|
+  # the pod url
+  config.server_uri = AppConfig.pod_uri
+
+  # the class to be used for a person
+  config.person_class = Person.to_s
```

Comment:
I think the accessor should duck type here and call `to_s` on its own. Assigning the actual classes is just too common, especially in the Rails world.